massa-execution/src/vm.rs

```diff
@@ -334,7 +336,13 @@ impl VM {
                 let public_key = derive_public_key(&private_key);
                 Address::from_public_key(&public_key)
             });
-            context.call_stack = vec![address].into();
+
+            context.stack = vec![StackElement {
+                address,
+                coins: AMOUNT_ZERO,
+                owned_addresses: vec![address].into_iter().collect(),
+            }]
+            .into();
```

Comment:
avoid this conversion by changing context.stack to be a vec instead of a vecdeque